MIGRATING.md

```diff
@@ -0,0 +1,84 @@
+# Okta Signin Widget migration guide
+
+This library uses semantic versioning and follows Okta's [library version policy](https://developer.okta.com/code/library-versions/). In short, we don't make breaking changes unless the major version changes!
+
+## Migrating from 2.x to 3.x
+
+### Consolidated css files to single one
+
+In version 2.x we had two separate css files to import: `okta-sign-in.min.css` and `okta-theme.css`. We moved to have one single css file which is still called `okta-sign-in.min.css`.
+
+Depending on your configuration, this is what you should change:
+
+- If you were using CDN links for the css, you will need to change the `okta-sign-in.min.css` link and remove the `okta-theme.css` link.
+
+Example:
+```html
+<!-- BEFORE -->
+<link
+  href="https://ok1static.oktacdn.com/assets/js/sdk/okta-signin-widget/2.14.0/css/okta-sign-in.min.css"
+  type="text/css"
+  rel="stylesheet"/>
+<link
+  href="https://ok1static.oktacdn.com/assets/js/sdk/okta-signin-widget/2.14.0/css/okta-theme.css"
+  type="text/css"
+  rel="stylesheet"/>
+
+<!-- AFTER -->
+  <link
+  href="https://ok1static.oktacdn.com/assets/js/sdk/okta-signin-widget/3.0.0/css/okta-sign-in.min.css"
+  type="text/css"
+  rel="stylesheet"/>
+```
+
+- If you were building your css files through `sass`, you will need to build it again and this time it will generate only `okta-sign-in.min.css` instead of the two files.
+
+### Renamed functions
+
+`tokenManager.refresh` has been renamed to `tokenManager.renew`, so you should update it in your code.
```

Comment:
nit: Let's keep the `signIn` object prefix to be more verbose:

```suggestion
`signIn.tokenManager.refresh` has been renamed to `signIn.tokenManager.renew`, so you should update it in your code.
```

Reply:
The developer could name the variable whatever they want, it isn't always `signIn`. I'd prefer to drop the prefix to stay in line with our API docs: https://github.com/okta/okta-signin-widget/#tokenmanagerrefresh